web/impact/impact/tests/test_analyze_judging_round_view.py

```diff
@@ -125,6 +125,16 @@ def test_get_with_industry_criterion(self):
         dists = _calc_industry_dists(options, judge_key)
         self.assert_option_distributions(context, dists)
 
+    def test_option_spec_option_field_can_be_nonempty(self):
```

Comment:
if im using the other test for reference, this should probably be `test_industry_option_field_can_be_nonempty`